gramps/plugins/db/dbapi/dbapi.py

```diff
@@ -628,6 +629,32 @@ def _commit_base(self, obj, obj_key, trans, change_time):
 
         return old_data
 
+    def _commit_raw(self, data, obj_key):
+        """
+        Commit the specified object to the database, storing the changes as
+        part of the transaction.
+        data is serialized primary object (must have handle in position 0)
+
+        This is intended to be use only for db upgrades.
+        """
+        table = KEY_TO_NAME_MAP[obj_key]
+        handle = data[0]
+
+        if self._has_handle(obj_key, handle):
+            # update the object:
```

Comment:
These two comments are not really necessary.

Reply:
done